### DISCUSSION TITLE
Turbo doesn't forward the AWS env vars by default

### DISCUSSION THREAD

**Discussion by nicholasgriffintn** [vercel/turborepo#8414] [Help] [answered]
### Summary

Maybe it should?

Commonly used and mean that commands that are using the AWS CLI expecting to use the existing AWS env vars (which are pre set in CodePipeline / CodeBuild) will fail without overriding the new default behaviour, or manually setting the keys.

**Comment by anthonyshew** [comment, marked as answer]
The intent is for you to set keys like this for your repository.
